parse.go

```diff
@@ -397,7 +402,7 @@ func getProtoFiles(root string, ignores string) ([]string, error) {
 func getUpdatedLock(ignores string) (*Protolock, error) {
 	// files is a map of filepaths to string buffers to be joined into the
```

Comment:
would you update this comment to reflect the change to a slice of ProtoFiles rather than the map implementation?